packages/cli/src/util/validate-config.ts

```diff
@@ -7,7 +7,7 @@ import {
   rewritesSchema,
   trailingSlashSchema,
 } from '@vercel/routing-utils';
-import { VercelConfig } from './types';
+import { VercelConfig } from './dev/types';
```

Comment:
The version in `dev/types` is just re-exported from client.

```suggestion
import { VercelConfig } from '@vercel/client';
```

Reply:
I found a few of these so I fixed in [af0e12d](https://github.com/vercel/vercel/pull/8622/commits/af0e12de588e5c8b18227d8328027223a700af24)